### DISCUSSION TITLE
next level - data servers?

### DISCUSSION THREAD

**Discussion by faraway20** [Chia-Network/chia-blockchain#6515] [Ideas]
hi all,

how you think to start chia mining with whis system:

CyberStore 490S Petabyte Storage Server
2X E5-2643 v4 Intel Xeon, 6 Cores, 3.40GHz, 20M Cache or better 2XE5-2699 v4 Intel Xeon, 22 Cores, 2.20GHz, 55Mb Cache ?
how i understand better core speed better plotting speed but more cores more plots plotting and the same time?
8X128GB 2933MHz DDR4 ECC Registered DIMM Module  so 1TB of ram (maybe its worth to create tempory RAM disk for plotting to save SSD endurance?)
2X3.2TB HGST Hitachi Enterprise-Class 2.5" 12Gb/s SAS SSD for tempory
90X18TB Enterprise Class SATA 7200RPM - 3.5" - Enterprise Class Drive so 1,47PiB storage space

Finaly system cost about 50eur/TB and 0,62eur/TB/month maintaince( space in tier 3 data center, electricity, internet)

**Comment by dungla2011** [comment]
It's too big, but the price chia now is low
USD/TiB Month = 4.x usd and down veryfast, so it's good for you?
